src/res/flame.html

```diff
@@ -15,16 +46,156 @@
 	#match {right: 0}
 	#reset {cursor: pointer}
 	#canvas {width: 100%; height: /*height:*/300px}
+
+	#info-tooltip {
+		display: none;
+		position: fixed;
+		width: 404px;
+		border: 1px solid #666666;
+		background: #ffffe0;
+		border-radius: 8px;
+		padding: 2px;
+		z-index: 1000;
+		pointer-events: none;
+	}
+	.top {
+		border-top: .1px solid #000
+	}
+	.left {
+		border-left: .1px solid #000
+	}
+	.bottom {
+		border-bottom: .1px solid #000
+	}
+	.right {
+		border-right: .1px solid #000
+	}
+	.toolbarIcon {
+		width: 24px;
+		height: 24px;
+		margin: 0 2px;
+		display: flex;
+		justify-content: center;
+		align-items: center;
+		cursor: pointer;
+	}
+	.toolbarSelected {
+		background-color: #cccccc;
+		outline: 1px solid #999999;
+		border-radius: 4px;
+	}
+	.toolbarIcon:hover {
+		background-color: #ffffe0;
+		outline: 1px solid #ffc000;
+		border-radius: 4px;
+	}
+	.bordered {
+		outline: none;
+		border: none;
+	}
+	.colortip {
+		width: 200px;
+		box-sizing: border-box;
+		padding: 2px;
+		margin: 1px 0;
+	}
+	.colortip-title {
+		font-weight: bold;
+		padding: 2px;
+	}
+	.canvas-context-menu {
+		position: fixed;
+		background-color: white;
+		border: 1px solid #ccc;
+		box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.2);
+		border-radius: 4px;
+		padding: 2px 0;
+		min-width: 150px;
+		z-index: 1000;
+		display: none;
+	}
+	.menu-item {
+		padding: 2px 4px;
+		cursor: pointer;
+	}
+	.menu-item:hover {
+		background-color: #ffc000;
+	}
+	.menu-separator {
+		height: 1px;
+		background-color: #ddd;
+		margin: 2px 0;
+	}
 </style>
 </head>
 <body style='font: 12px Verdana, sans-serif'>
-<h1>/*title:*/</h1>
-<header style='text-align: left'><button id='inverted' title='Invert'>&#x1f53b;</button>&nbsp;&nbsp;<button id='search' title='Search'>&#x1f50d;</button></header>
-<header style='text-align: right'>Produced by <a href='https://github.com/async-profiler/async-profiler'>async-profiler</a></header>
-<canvas id='canvas'></canvas>
-<div id='hl'><span></span></div>
-<p id='status'></p>
-<p id='match'>Matched: <span id='matchval'></span> <span id='reset' title='Clear'>&#x274c;</span></p>
+
+<div class="title-toolbar-container">
+	<div class="left-toolbar">
+		<div id="graph-info" class="toolbarIcon"><svg width="20" height="20" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><circle cx="10" cy="10" r="8" stroke="#004d80" stroke-width="1" fill="none"/><path d="M10 5.5C8.75 5.5 7.75 6.5 7.75 7.75H9C9 7.06 9.56 6.5 10.25 6.5C10.94 6.5 11.5 7.06 11.5 7.75C11.5 8.4 10.95 8.75 10.5 8.95C9.8 9.3 9.25 9.8 9.25 10.75V11H10.75V10.75C10.75 10.38 11.04 10.1 11.43 9.92C12.16 9.58 12.75 9.05 12.75 7.75C12.75 6.5 11.25 5.5 10 5.5Z" fill="#ff8d40" stroke="#ff8d40" stroke-width="0.6" stroke-linecap="round" stroke-linejoin="round"/><circle cx="10" cy="13.5" r="1" fill="#ff8d40"/></svg></div>
```

Comment:
WDYT about changing order of icons to match previous layout? E.g., Invert button first, then Search, and Help in the end.